app/assets/javascripts/models/beta/iteration.js

```diff
@@ -53,53 +53,93 @@ export const getIterationForStory = (story, project) => {
   }
 };
 
-const createSprint = (sprintNumber = 0, startDate = 0, isFiller = false, velocity) => ({
+const createSprint = (
+  sprintNumber = 0, 
+  startDate = 0, 
+  isFiller = false, 
+  velocity, 
+  stories = [],
+  points = 0,
+  remainingPoints = calculateRemainingPoints(velocity, points)
+) => ({
   number: sprintNumber,
   startDate,
-  points: 0,
+  points,
   completedPoints: null,
-  stories: [],
+  stories,
   isFiller,
-  remainingPoints: velocity
+  remainingPoints
 });
 
-const createFillerSprints = (size, initialNumber, project, velocity) => {
-  return _.times(size, (i) => {
+export const calculateRemainingPoints = (velocity, points) =>
```

Comment:
Is this export really needed? :thinking:

Reply:
Yep, because of the tests here: https://github.com/Codeminer42/cm42-central/pull/593/files#diff-713c0b32b616b0cc1b112f661cd3f30dR5